invisible_cities/conftest.py

```diff
@@ -378,43 +378,43 @@ def KrMC_hdst(ICDATADIR):
              7, 9, 2, 7, 2, 4]
 
     X     = [137.02335083, 136.05361214, 155, 137.50739563,
-             139.44340629, 135, 0.,
+             139.44340629, 135, NN,
```

Comment:
I don't doubt that this is appropriate, but I am curious what the intention is here.  It seems odd to me to mix numbers and NN in lists like this.  I can understand if there is a concern about divide by zeroes or something I suppose.  What is the purpose of this change?

Reply:
`NN` is not actually a `nan` as you might expect. It's defined as `-99999` or something like that. I missed this part of the development but I think the reason to use `NN` and not `nan` was related to I/O. I plan to revisit this in the near future, but for now, it will stay like this.
As for your actual question, I replaced 0 with NN because 0 is a realistic value that you might not notice when analyzing data. For instance, if you plot a bunch of events and only a few of them have `0` you will not see anything weird. If they have `-99999` you surely will.

Reply:
I remember discussing this many moons ago, and being unconvinced, but I don't recall the details at all. Here is the [relevant commit](https://github.com/next-exp/IC/commit/1050194986d6ac1d2a897b9a3bc1d3c9d5a166a9) with a message which offers a little explanation, but not enough for me understand exactly what problem it is supposed to solve and how.

The commit was part of #272, but a quick glance at the discussion doesn't reveal any deep insights.

Reply:
Thanks, this is what I vaguely remembered. I will try to dig deeper when I finally tackle this.